model/riscv_pmp_regs.sail

```diff
@@ -66,6 +66,9 @@
 /*  SUCH DAMAGE.                                                                         */
 /*=======================================================================================*/
 
+function haveSmepmp() -> bool =
+  return false
```

Comment:
No need to return, and can just be a single line

Reply:
Also why isn't this with the other haveFoo functions?

Reply:
This is added temporarily and will not be the part of the final implementation. Just for checking purposes.